torchio/data/image.py

```diff
@@ -124,7 +126,7 @@ class Image(dict):
     """
     def __init__(
             self,
-            path: Optional[TypePath] = None,
+            path: Union[TypePath, Sequence[TypePath]] = None,
```

Comment:
If `path` can be `None`, this should be `Optional[TypePath, Union[TypePath, Sequence[TypePath]]]` or `Union[TypePath, Sequence[TypePath], None]`. I think the latter is more readable.